src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.task.Task;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
+
+    @Test
+    public void todoTest() {
+        try {
+            Task task = Task.createTask("Read a book", "todo", "", "");
+            assertEquals("T", task.getType());
+            assertEquals("Read a book", task.getName());
+            assertEquals("[T][ ] Read a book", task.toString());
+            assertEquals("T | 0 | Read a book", task.toSaveFormat());
+        } catch (DukeException ex) {
+            assertEquals(0, 1); // force fail
```

Comment:
I think this is one way. But there are also ways to assert for exceptions. Alternatively you can simply 
public void todoTest() throws DukeException() {...}
I think